README.Rmd

```diff
@@ -147,6 +147,10 @@ To reproduce book **in its entirety**, run the following command (which installs
 ```{r readme-install-github-2, message=FALSE, results='hide'}
 #| eval=FALSE
 # To reproduce all chapters (install lots of packages, may take some time!)
+options(repos = c(
+  geocompx = 'https://geocompx.r-universe.dev',
+  cran = 'https://cloud.r-project.org'
+))
 remotes::install_github("geocompr/geocompkg", dependencies = TRUE)
```

Comment:
In this case we can just use `install.packages("geocompkg", dependencies = TRUE)` instead of `remotes::install_github("geocompr/geocompkg", dependencies = TRUE)`

Reply:
@Robinlovelace

Reply:
Yes. Suggestion: we offer both alternatives for people who don't want to update their repos.

BTW I get notifications on this thread regardless of whether or not I'm tagged.